packages/teleport/src/Welcome/NewCredentials/NewMfaDevice.tsx

```diff
@@ -66,6 +70,11 @@ export function NewMfaDevice(props: Props) {
   ) {
     e.preventDefault(); // prevent form submit default
 
+    userEventService.capturePreUserEvent({
+      event: userEvents.preUser.onboard.registerChallengeSubmitEvent,
```

Comment:
should we also capture this for `NewPasswordlessDevice`?

Reply:
Added to the list for a follow-up PR!